homeassistant/components/ibeacon/coordinator.py

```diff
@@ -354,7 +354,25 @@ def _async_check_unavailable_groups_with_random_macs(self) -> None:
             for group_id in self._group_ids_random_macs
             if group_id not in self._unavailable_group_ids
             and (service_info := self._last_seen_by_group_id.get(group_id))
-            and now - service_info.time > UNAVAILABLE_TIMEOUT
+            and (
+                # We will not be callbacks for iBeacons with random macs
```

Comment:
will not add callbacks

Reply:
https://github.com/home-assistant/core/pull/82718